src/ebmc/bdd_engine.cpp

```diff
@@ -163,8 +163,8 @@ int bdd_enginet::operator()()
       std::cout << "Atomic propositions:\n";
       for(const auto & a : atomic_propositions)
       {
-        std::string as_string=
-          from_expr(namespacet(symbol_table), irep_idt(), a.first);
```

Comment:
Just because I am reading this for the first time: should likely use `format` instead of `from_expr`: the empty `irep_idt` will make it fall back to the default formatter, which is C.

Reply:
#111